### PR TITLE
MOTECH-2536 MOTECH-2537: Fixed question labels in commcare forms

### PR DESCRIPTION
**Pull request by weronika-redlarska**
Connected with pull request: https://github.com/motech/modules/pull/289
Test for blank question label in form added.

### CODE REVIEW

File: commcare/src/test/java/org/motechproject/commcare/tasks/builder/DisplayNameHelperTest.java [hunk 1]
```diff
@@ -20,4 +20,5 @@ public void shouldBuildDisplayNamesWithFourArguments() {
     public void shouldBuildDisplayNamesWithTwoArguments() {
         assertEquals("User [myConf]", DisplayNameHelper.buildDisplayName("User", "myConf"));
     }
+
```
Comment (sebbrudzinski) [inline commcare/src/test/java/org/motechproject/commcare/tasks/builder/DisplayNameHelperTest.java]: remove

File: commcare/src/test/java/org/motechproject/commcare/tasks/builder/FormActionBuilderTest.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+        when(schemaService.retrieveApplications("ConfigOne")).thenReturn(DummyCommcareSchema.getApplicationsWithCustomQuestionLabel(displayName));
+        List<ActionEventRequest> actionEventRequests = formActionBuilder.buildActions();
+
+        assertEquals(actionEventRequests.get(0).getActionParameters().first().getDisplayName().length(), 255);
+        assertEquals(actionEventRequests.get(0).getActionParameters().first().getDisplayName(), shortenedDisplayName);
+    }
+
+    @Test
+    public void shouldNotAllowActionParametersDisplayNameBlank() {
```
Comment (sebbrudzinski) [inline commcare/src/test/java/org/motechproject/commcare/tasks/builder/FormActionBuilderTest.java]: Rename to "shouldUseQuestionIdAsDisplayNameIfQuestionLabelIsEmpty"

**Comment by pgesek** [comment]
Can we close this?

**Comment by sebbrudzinski** [comment]
No, this adds a test for blank question labels, but needs to be merged with master first.

**Pull request merged**